go/extra/stats/cmd/stats.go

```diff
@@ -293,13 +293,7 @@ func getStats(ctx context.Context, consensus consensusAPI.ClientBackend, registr
 		}
 
 		// Go over all signatures for a block.
-		// XXX: In tendermint master (not yet released) the signatures are
-		// obtained in LastCommit.Signatures.
-		for _, sig := range tmBlockMeta.LastCommit.Precommits {
-			if sig == nil {
```

Comment:
Should this skip if `sig.Absent()` is true?

Reply:
Dunno, not my tool.

Reply:
Done.